locales/en.json

```diff
@@ -1,10 +1,10 @@
 {
     "Title": "Contoso Air",
 	"Home": {
-		"HeroMessage": "Where do you<br>want to go?"
+		"HeroMessage": "Feed your heart"
```

Comment:
This isn't clear enough.  Let's do. "Fly. Feed your heart."

Reply:
That sounds good.